cmd/transform/transform.go

```diff
@@ -36,21 +34,19 @@ func main() {
 Examples:
 
 # Execute with filename
-$GOPATH/bin/transform -transformers=URLTransformer,AMPBoilerplateTransformer \
-   /path/to/input.html
+$GOPATH/bin/transform /path/to/input.html
 
 # Execute with pipe
-cat /path/to/input.html | $GOPATH/bin/transform -transformers=URLTransformer,AMPBoilerplateTransformer
+cat /path/to/input.html | $GOPATH/bin/transform
 `)
 	}
 
 	flag.Parse()
-	var transformers []string
-	if *transformersFlag != "" {
-		transformers = strings.Split(*transformersFlag, ",")
-		for i := range transformers {
-			transformers[i] = strings.TrimSpace(transformers[i])
-		}
+	transformers := []string{
```

Comment:
(Eventually we'll probably want this somewhere that library-users can get it, too.)